datafusion/macros/src/user_doc.rs

```diff
@@ -192,13 +185,13 @@ pub fn user_doc(args: TokenStream, input: TokenStream) -> TokenStream {
     let input = parse_macro_input!(input as DeriveInput);
     let name = input.clone().ident;
 
-    let doc_section_include: bool = if let Some(doc_section_include) = doc_section_include
-    {
-        doc_section_include.value().parse().unwrap()
-    } else {
-        true
-    };
-
+    let label = doc_section_lbl.as_ref().unwrap().value();
+    let doc_section_option = doc_sections_const().iter().find(|ds| ds.label == label);
```

Comment:
lets document this part, mentioning we trying to find a predefined const by `label` field